.github/workflows/test.yml

```diff
@@ -92,29 +85,17 @@ jobs:
           python -m pip install -r requirements.txt
 
       - name: Run pytest
-        run: python -m pytest --pyargs curated_transformers --slow
+        run: python -m pytest --pyargs curated_transformers
```

Comment:
Removing the cache sounds great, but maybe we want to keep the slow markers as a middle ground for the future? Or maybe rename the existing `veryslow` tests to `slow`.

Reply:
Can't argue with that. I'll rename `veryslow` to `slow` and update the BuildKite pipeline.